packages/error-reporting/index.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
Can you use `/*!` here? We use that syntax to tell the documentation to ignore this comment block.